engine/src/test/java/com/redhat/cloud/notifications/templates/TestInventoryTemplate.java

```diff
@@ -130,6 +130,8 @@ public void testDailyEmailBody(final String groupToValidte) {
         assertTrue(result.contains("Host Name"), "Body should contain 'Host Name' header");
         assertTrue(result.contains("Error"), "Body should contain 'Error' header");
         assertTrue(result.contains(TestHelpers.HCC_LOGO_TARGET));
+        // TODO set boolean to false once app-specific daily digest code is removed
```

Comment:
Based on [this comment](https://github.com/RedHatInsights/notifications-backend/pull/3222#discussion_r2028934059), this should be set to true for now.